backend/src/test/scala/cromwell/backend/standard/callcaching/BlacklistCacheSpec.scala

```diff
@@ -1,22 +1,24 @@
 package cromwell.backend.standard.callcaching
 
 import cromwell.core.CacheConfig
+import cromwell.services.CallCaching.CallCachingEntryId
 import org.scalatest.{BeforeAndAfterAll, FlatSpec, Matchers}
 
 import scala.concurrent.duration._
 import scala.language.postfixOps
 
 class BlacklistCacheSpec extends FlatSpec with BeforeAndAfterAll with Matchers {
+  val hit = CallCachingEntryId(3)
```

Comment:
Seems a shame to get this far but then not test the `blacklistHit` functionality...

Reply:
For this little workup I only did enough to get the production code looking reasonable, but yes the hit blacklisting should be tested. 🙂